sdk/keyvault/ci.yml

```diff
@@ -1,4 +1,9 @@
 # NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+parameters:
+- name: RunLiveTests
+  displayName: 'Run live tests'
+  type: boolean
+  default: false
```

Comment:
Why is this defined here or in any service directory? It should be in the template. All services should run live tests (if they have any).

Reply:
It can't be surfaced to users running the pipeline manually if it's not in the outermost pipeline yaml:

![image](https://github.com/user-attachments/assets/3705232c-0f5d-4f95-a7d5-80d0f62d7ced)

Also, I thought we said that all services should run live tests for scheduled runs and should enable live tests for manual runs when queuing the build:

```yaml
RunLiveTests: ${{ or(parameters.RunLiveTests, eq(variables['Build.Reason'], 'Schedule')) }}
```

Reply:
I don't see this in any other language and they work the way I think makes sense. `net - keyvault - tests` run live tests by default. I can queue it manually as well. `sdk/keyvault/ci.yml` does not have a variable defined to do it.

The minimal amount of authoring *required* should be in service directories' `ci.yml` files or people are going to screw things up. We should dictate the default behavior, not make them opt into it or even knowing they have to opt into it.

Reply:
we don't have `rust - keyvault - tests`  because we use unified (ci + test) pipelines, so [go - azsecrets](https://dev.azure.com/azure-sdk/internal/_build?definitionId=6440) would be the analog here.

It's a single pipeline for CI, Nightly, Manual and Release
![image](https://github.com/user-attachments/assets/8f74982a-7bd6-4afb-8f77-3c83476e930d)

To manually run tests on a branch, just queue the single pipeline, change the ref to build and don't approve the release stages

Reply:
What we discussed was being able to turn off live tests for release runs.  Because a release run is just a manual invocation of the single pipeline, there's no way to know, without a queue time parameter, if someone is queuing it because they want to release or queuing it because they want to run live tests.

Reply:
Discuss offline.